src/components/common/YourCompany/YourCompany.vue

```diff
@@ -7,16 +7,18 @@
 
     <div class="define-company-header">
       <v-icon color="appDkBlue">mdi-domain</v-icon>
-      <label class="define-company-title">Your Company</label>
+      <label class="define-company-title">Your {{ getResource.entityReference }}</label>
     </div>
 
     <div id="company-name-section" class="section-container" :class="{'invalid-section': invalidNameSection}">
       <!-- Company Name -->
       <v-row no-gutters class="mt-4">
         <v-col cols="3">
-          <label :class="{'error-text': invalidNameSection}"><strong>Company Name</strong></label>
+          <label :class="{'error-text': invalidNameSection}">
+            <strong>{{ getResource.entityReference }} Name</strong>
```

Comment:
Oh, you use it here, too.

Well, how about a new resource string for this?

Reply:
For sure!

Reply:
Or maybe a getter, if the rule for company vs business is clear?

Reply:
Same as above, i've left this resource value as the single reference. 'company' or 'business' etc are the 1 commonality between the designs requirements where these values are used, with minor adjustments between implementations. 
`Your Company` and `Your Business` for one example, `Company Name` and `Business Name` for another example.
I think it makes sense to make the common value and resource that alone. 
Going to leave as is 👍

Reply:
ok